src/axes.ts

```diff
@@ -607,6 +607,13 @@ export class Axis extends Shape {
       maxValue + valueRange * this.marginRatio];
   }
 
+  public setRubberbandRange(minValue: string, maxValue: string): void {
+    if (!this.isDiscrete) {
+      this.rubberBand.minValue = Math.min(Number(minValue), Number(maxValue));
```

Comment:
In the case you entered a wider value on the left. I guess front app manage it, but html of debug mode don't. So I can remove it if you want, because it is essentially a user bug and not an app bug